multicluster/build/Dockerfile.build

```diff
@@ -18,4 +18,4 @@ LABEL description="The Docker image to deploy the Antrea Multicluster controller
 
 USER root
 
-COPY --from=antrea-build /antrea/multicluster/bin/* /usr/local/bin/
```

Comment:
In antrea's own build directory, the dockerfile is put under build/images/, could we make them consistent?

Reply:
done.